Kickstarter-iOS/Features/PledgedProjectsOverview/PPOContainerViewController.swift

```diff
@@ -20,6 +23,26 @@ public class PPOContainerViewController: PagedContainerViewController<PPOContain
       (.projectAlerts(.count(5)), ppoViewController),
       (.activityFeed(.dot), activitiesViewController)
     ])
+
+    let tabBarController = self.tabBarController as? RootTabBarViewController
+
+    self.ppoViewModel.$results.sink { [weak self] results in
+      let badge: TabBarBadge = results.values.count > 0 ? .count(results.values.count) : .none
+      self?.setPagedViewControllers([
+        (.projectAlerts(badge), ppoViewController),
+        (.activityFeed(.dot), activitiesViewController)
```

Comment:
The activity feed count, however, is one that we can (and should) get. I think it's stored on the user object. If you can't find it, let me know, and I'll help look! (We should be using it for the tabbar count when the PPO flag is off.) If this gets complicated, it might make more sense to do the badges in a separate pr.

Reply:
I added a new PPOContainerViewModel and added activity feed badge in https://github.com/kickstarter/ios-oss/pull/2161/commits/549d151211a739f3b273fd6d6217aeff25c30526 using similar patterns for handling user updates I'd seen elsewhere in the app.